datameta/api/apikeys.py

```diff
@@ -51,7 +51,7 @@ def __json__(self, request: Request) -> dict:
                 "email": self.email,
                 "token": self.token,
                 "label": self.label,
-                "expiresAt": self.expires_at.isoformat() if self.expires_at else None
```

Comment:
We cannot return the expires_at directly, as it is a datetime object, however, openapi expects string as a response. I could also suggest to format the datetime object in isoformat to stay consistent.